project_script/main.py

```diff
@@ -1,108 +1,105 @@
-import os.path
-import numpy as np
 import pandas as pd
 import matplotlib.pyplot as plt
 from scipy import integrate
 import ask
-
-
-def separate_even_columns(data_frame):
-    return data_frame.iloc[:, [i for i in range(len(data_frame.columns)) if i % 2 == 1]]
-
-
-def separate_odd_columns(data_frame):
-    return data_frame.iloc[:, [i for i in range(len(data_frame.columns)) if i % 2 == 0]]
-
-
-def is_valid_file(file_path):
-    return True if file_path != "" and os.path.exists(file_path) else False
+import datatreatment as dt
+import constantvalues as cv
 
 
 def message_invalid_path():
     return print("Arquivo inválido ou inexistente \n")
 
 
-def is_bigger_data_frame(data_frame_1, data_frame_2):
-    return len(data_frame_1) > len(data_frame_2)
-
-
-def equalize_data_frame_rows(data_frame_1, data_frame_2):
-    return data_frame_1.loc[0:len(data_frame_2)-1, :]
-
-
 def sanitize_data_frames():
     global even_columns_subtracted, time_photo_celiv_ramp_time
-    if is_bigger_data_frame(time_photo_celiv_ramp_time, even_columns_subtracted):
-        time_photo_celiv_ramp_time = equalize_data_frame_rows(time_photo_celiv_ramp_time, even_columns_subtracted)
+    if dt.is_bigger_data_frame(
+            time_photo_celiv_ramp_time,
+            even_columns_subtracted
+    ):
+        time_photo_celiv_ramp_time = dt.equalize_data_frame_rows(
+            time_photo_celiv_ramp_time,
+            even_columns_subtracted
+        )
         return
-    even_columns_subtracted = equalize_data_frame_rows(even_columns_subtracted, time_photo_celiv_ramp_time)
+    even_columns_subtracted = dt.equalize_data_frame_rows(
+        even_columns_subtracted,
+        time_photo_celiv_ramp_time
+    )
 
 
 def integrate_data():
     global time_photo_celiv_ramp_time_transposed, even_columns_subtracted_transposed_in_arrays
     integrated_values = []
     for key, value in enumerate(even_columns_subtracted_transposed_in_arrays):
-        integrated_values.append(integrate.simps(even_columns_subtracted_transposed_in_arrays[key],
-                                                 time_photo_celiv_ramp_time_transposed[key], even='avg'))
+        integrated_values.append(
+            integrate.simps(
+                even_columns_subtracted_transposed_in_arrays[key],
+                time_photo_celiv_ramp_time_transposed[key],
+                even='avg'
+            )
+        )
     return integrated_values
 
 
-# device_thickness = ask.ask_device_thickness()
+# device_thickness = ask.device_thickness()
+# device_area = ask.device_area()
+charge_density_calculation_constant = cv.current_correction_factor * cv.charge_density_correction_factor \
```

Comment:
The name `_constant` is it to flag that variable it's a constant, or it's something about physics? If it's to flag, i made a comment about constants above, but if the answer is no, just ignore.